<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Grid">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>641</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/Connect4-Grid.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Col0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Col7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Col 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cell70">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell71">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell72">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell73">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell74">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell75">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell76">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell77">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell60">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell50">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell40">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell30">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell20">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell00">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell01">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell02">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell03">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell04">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell05">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell06">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell07">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell14">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell15">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell16">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell17">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell22">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell23">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell24">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell25">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell26">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell27">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell31">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell32">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell33">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell34">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell35">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell36">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell37">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell41">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell42">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell43">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell44">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell45">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell46">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell47">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell51">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell52">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell53">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell54">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell55">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell56">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell57">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Cell61">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell62">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell63">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell64">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell65">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell66">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Cell67">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoText">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;P1's Turn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="turnText">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bernard MT Condensed</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Turn #1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>Col0</zorder>
   <zorder>Col1</zorder>
   <zorder>Col2</zorder>
   <zorder>Col3</zorder>
   <zorder>Col4</zorder>
   <zorder>Col5</zorder>
   <zorder>Col6</zorder>
   <zorder>Col7</zorder>
   <zorder>Cell70</zorder>
   <zorder>Cell71</zorder>
   <zorder>Cell72</zorder>
   <zorder>Cell73</zorder>
   <zorder>Cell74</zorder>
   <zorder>Cell75</zorder>
   <zorder>Cell76</zorder>
   <zorder>Cell77</zorder>
   <zorder>Cell60</zorder>
   <zorder>Cell50</zorder>
   <zorder>Cell40</zorder>
   <zorder>Cell30</zorder>
   <zorder>Cell20</zorder>
   <zorder>Cell10</zorder>
   <zorder>Cell00</zorder>
   <zorder>Cell01</zorder>
   <zorder>Cell02</zorder>
   <zorder>Cell03</zorder>
   <zorder>Cell04</zorder>
   <zorder>Cell05</zorder>
   <zorder>Cell06</zorder>
   <zorder>Cell07</zorder>
   <zorder>Cell11</zorder>
   <zorder>Cell12</zorder>
   <zorder>Cell13</zorder>
   <zorder>Cell14</zorder>
   <zorder>Cell15</zorder>
   <zorder>Cell16</zorder>
   <zorder>Cell17</zorder>
   <zorder>Cell21</zorder>
   <zorder>Cell22</zorder>
   <zorder>Cell23</zorder>
   <zorder>Cell24</zorder>
   <zorder>Cell25</zorder>
   <zorder>Cell26</zorder>
   <zorder>Cell27</zorder>
   <zorder>Cell31</zorder>
   <zorder>Cell32</zorder>
   <zorder>Cell33</zorder>
   <zorder>Cell34</zorder>
   <zorder>Cell35</zorder>
   <zorder>Cell36</zorder>
   <zorder>Cell37</zorder>
   <zorder>Cell41</zorder>
   <zorder>Cell42</zorder>
   <zorder>Cell43</zorder>
   <zorder>Cell44</zorder>
   <zorder>Cell45</zorder>
   <zorder>Cell46</zorder>
   <zorder>Cell47</zorder>
   <zorder>Cell51</zorder>
   <zorder>Cell52</zorder>
   <zorder>Cell53</zorder>
   <zorder>Cell54</zorder>
   <zorder>Cell55</zorder>
   <zorder>Cell56</zorder>
   <zorder>Cell57</zorder>
   <zorder>Cell61</zorder>
   <zorder>Cell62</zorder>
   <zorder>Cell63</zorder>
   <zorder>Cell64</zorder>
   <zorder>Cell65</zorder>
   <zorder>Cell66</zorder>
   <zorder>Cell67</zorder>
   <zorder>Grid</zorder>
   <zorder>infoText</zorder>
   <zorder>turnText</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
